app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -69,154 +68,107 @@ const ruleFieldsConfig = {
   },
 };
 
-@injectIntl
-@connect(
-  (state) => ({
-    projectRole: activeProjectRoleSelector(state),
-    userRole: userAccountRoleSelector(state),
-    enabled: projectNotificationsEnabledSelector(state),
-    cases: projectNotificationsCasesSelector(state),
-    isEmailIntegrationAvailable: isEmailIntegrationAvailableSelector(state),
-  }),
-  {
-    updateNotificationsConfig: updateProjectNotificationsConfigAction,
-    showModal: showModalAction,
-  },
-)
-@track()
-export class Notifications extends Component {
-  static propTypes = {
-    intl: PropTypes.object.isRequired,
-    enabled: PropTypes.bool,
-    cases: PropTypes.array,
-    updateNotificationsConfig: PropTypes.func,
-    showModal: PropTypes.func,
-    projectRole: PropTypes.string,
-    userRole: PropTypes.string,
-    isEmailIntegrationAvailable: PropTypes.bool,
-    tracking: PropTypes.shape({
-      trackEvent: PropTypes.func,
-      getTrackingData: PropTypes.func,
-    }).isRequired,
-  };
-  static defaultProps = {
-    enabled: false,
-    cases: [],
-    showModal: () => {},
-    updateNotificationsConfig: () => {},
-    projectRole: '',
-    userRole: '',
-    isEmailIntegrationAvailable: true,
-  };
+export const Notifications = ({ updateNotificationsConfig }) => {
```

Comment:
Why you pass it outside if it's should be `updateProjectNotificationsConfigAction`? you can use dispatch it directly in the component
Also you have passed nothing to here in `ProjectSettingsPageContainer`